firmware/ec/platform/oc-sdr/cfg/OC_CONNECT_FE.c

```diff
@@ -42,27 +42,31 @@ Eeprom_Cfg eeprom_fe_inv = {
 // FE Channel 1 Power sensor.
 INA226_Dev fe_ch1_ps_5_7v = {
     /* CH1 5.7V Sensor */
-    .cfg = {
-        .dev = {
-            .bus = OC_CONNECT1_I2C4,
-            .slave_addr = RFFE_INA226_CH1_5_7V_ADDR,
-        },
-        .pin_alert = &(OcGpio_Pin){ &ec_io,
-                                    OC_EC_RFFE_TEMP_INA_ALERT },
-    },
+    .cfg =
```

Comment:
I am not sure which rule is causing the change of indents , if its possible can we have it as it was before . 
Because we will have 3 or 4 levels of braces , which can cause multiple unwanted indents .

Reply:
Are you referring to the larger indents? That was some of the settings I missed like the one you mentioned in the above comment. Updated those and back to indents of 4.